src/repositories/versionable/ IVersionableDocument.ts

```diff
@@ -0,0 +1,11 @@
+import * as mongoose from 'mongoose';
+
+export default interface IVersionableDocument extends mongoose.Document {
+    originalId: string;
+    createdAt: Date;
+    deletedAt: Date;
+    deletedBy: string;
+    createdBy: string;
+    updatedAt: Date;
+    updatedBy: string;
```

Comment:
updatedAt ,updatedBy, createdBy, deletedBy fields are not defined in VersionableSchema, So if not in use remove them or define VersionableSchema.